src/main/java/io/ddavison/conductor/Locomotive.java

```diff
@@ -201,21 +223,32 @@ public Locomotive() {
     static {
         // Set the webdriver env vars.
         if (JvmUtil.getJvmProperty("os.name").toLowerCase().contains("mac")) {
-            System.setProperty("webdriver.chrome.driver", findFile("chromedriver.mac"));
+            System.setProperty(ChromeDriverService.CHROME_DRIVER_EXE_PROPERTY, findFile("chromedriver.mac"));
         } else if (JvmUtil.getJvmProperty("os.name").toLowerCase().contains("nix") ||
```

Comment:
"os.name" and "nix" should be static field and used instead new String creation each time.